### ISSUE TITLE
v3.3/glfw: port `GLFW_CONTEXT_NO_ERROR`

### ISSUE THREAD

**Issue by hajimehoshi** [go-gl/glfw3#353]
https://www.glfw.org/docs/3.3/group__window.html#ga5a52fdfd46d8249c211f923675728082

Apparently `ContextNoError` is not defined

**Comment by dmitshur** [comment]
Yep, looks like it hasn't been added. Please feel free to send a PR if you have a need for it.
